qiskit/aqua/operators/list_ops/summed_op.py

```diff
@@ -132,6 +134,21 @@ def reduce(self) -> OperatorBase:
         else:
             return cast(OperatorBase, reduced_ops)
 
+    def to_circuit(self) -> QuantumCircuit:
+        """ Returns the quantum circuit, representing the summed operator.
+        Returns:
+            The circuit representation of the summed operator.
+        Raises:
+            ExtensionError: if operator is not unitary
```

Comment:
You don't need to add this here since it is not this method that raises this error, but some other function that is called inside of this
```suggestion
```

Reply:
Yes, but it warns user that operator needs to be unitary to use that method.
So maybe it is useful to have it in documentation?